Makefile

```diff
@@ -238,6 +238,9 @@ $(OUT)/treasuryDAO_storage.tz: src/**
 	# See "$(OUT)/treasuryDAO_storage.tz" for compilation result #
 	#
 
+metadata: all
+	stack exec -- baseDAO-ligo-meta print-metadata
```

Comment:
I think this should use `$(MAKE) -C haskell` and not `stack` directly (see `test` below for example)

Reply:
I had to modify `haskell/Makefile` to do this, I pushed the update now.